docs/modules/ROOT/pages/user-code-deployment.adoc

```diff
@@ -20,7 +20,9 @@ Below are the configuration options for the User Code Deployment feature.
 |JAR files from an external bucket are placed under the Java classpath when the following parameter values are supplied:
 
   - `secret`: Name of the Secret object which holds the credentials for your cloud provider.
-  - `bucketURI`: Full path for the external bucket. For example: `gs://your-bucket/path/to/jars`. 
+  - `bucketURI`: Full path for the external bucket. For example: `gs://your-bucket/path/to/jars`.
+
+WARNING: Currently, locally mounted drives are NOT supported for `bucketURI`; only cloud provider external buckets are supported.
```

Comment:
```suggestion
WARNING: Currently, mounted PVs are not supported; only cloud provider external buckets are supported.
```

Could you move this warning to another place because it is not related `bucketURI`? `bucketURI` is external bucket specific term.